app/jobs/send_daily_alert_email_job.rb

```diff
@@ -11,11 +11,11 @@ def perform
 
       Rails.logger.info("Sidekiq: Sending vacancy alerts for #{vacancies.count} vacancies")
 
-      AlertMailer.daily_alert(s.id, vacancies.pluck(:id)).deliver_later
+      AlertMailer.daily_alert(s.id, vacancies.pluck(:id)).deliver_later(queue: :email_daily_alerts)
```

Comment:
Sweet - can we get a test in for this to make sure the job gets sent to the right queue?

Reply:
Quite right 👍